scripts/build-release-artifacts.sh

```diff
@@ -24,8 +24,9 @@ set -o nounset
 set -o pipefail
 
 RELEASE_TAG="${1-}"
-if [[ ! "${RELEASE_TAG}" ]]; then
-  >&2 echo "usage: $0 <release tag of the form v[0-9]+.[0-9]+.[0-9]+>"
+RELEASE_TAG_REGEX="^v[0-9]+\.[0-9]+\.[0-9]+(-rc[0-9]+)?$"
```

Comment:
Would it make sense for this regex and check be defined once (e.g. in `util.sh`) rather than being duplicated across scripts?